docs/tasks/administer-cluster/configure-pod-disruption-budget.md

```diff
@@ -40,13 +42,37 @@ disruption budget to be violated.
 
 ## Specifying a PodDisruptionBudget
 
-A `PodDisruptionBudget` has two components: a label selector `selector` to specify the set of
-pods to which it applies, and `minAvailable` which is a description of the number of pods from that
+A `PodDisruptionBudget` has three components, of which two must be specified: 
```

Comment:
... has three components: one required, and another two, exactly one of which must be set.